package.json

```diff
@@ -19,6 +19,7 @@
   "files": [
     "bin/**/*",
     "dist/**/*.js?(.map)",
+    "dist/**/*.json",
```

Comment:
this will include all json files, is this "safe"/what we want?

Reply:
hmm, good question.  maybe, but the package.json is the only json file so we might as well be explicit.

Reply:
you know my motto!